app/src/sharedTest/java/org/oppia/app/topic/conceptcard/ConceptCardFragmentTest.kt

```diff
@@ -25,12 +29,40 @@ import javax.inject.Singleton
 @RunWith(AndroidJUnit4::class)
 class ConceptCardFragmentTest {
 
+  @get:Rule
+  val activityTestRule = ActivityTestRule(ConceptCardFragmentTestActivity::class.java)
+
   @Test
-  fun testConceptCardFragment_loadFragment_textIsDisplayed() {
-    // I'm not sure how to launch just the fragment because it doesn't have its own activity
-    ActivityScenario.launch(HomeActivity::class.java).use {
-      onView(withId(R.id.rich_text_card)).check(matches(withText("Hello World")))
-    }
+  fun testConceptCardFragment_openDialogFragmentWithSkill1_explanationIsDisplayed() {
+    onView(withId(R.id.open_dialog_1)).perform(click())
+    onView(withId(R.id.explanation)).check(matches(withText("Explanation with <b>rich text</b>.")))
+  }
+
+  @Test
+  fun testConceptCardFragment_openDialogFragmentWithSkill1_workedExamplesAreDisplayed() {
+    onView(withId(R.id.open_dialog_1)).perform(click())
+    onView(withText("Worked example with <i>rich text</i>.")).check(matches(isDisplayed()))
+  }
+
+  @Test
+  fun testConceptCardFragment_openDialogFragmentWithSkill2_explanationIsDisplayed() {
+    onView(withId(R.id.open_dialog_2)).perform(click())
+    onView(withId(R.id.explanation)).check(matches(withText("Explanation without rich text.")))
+  }
+
+  @Test
+  fun testConceptCardFragment_openDialogFragmentWithSkill2_workedExamplesAreDisplayed() {
+    onView(withId(R.id.open_dialog_2)).perform(click())
+    onView(withText("Worked example without rich text.")).check(matches(isDisplayed()))
+    onView(withText("Second worked example.")).check(matches(isDisplayed()))
+  }
+
+  @Test
+  fun testConceptCardFragment_openDialogFragmentWithSkill2_afterConfigurationChange_workedExamplesAreDisplayed() {
+    onView(withId(R.id.open_dialog_2)).perform(click())
+    activityTestRule.activity.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE
```

Comment:
Does this work as-is or do you also need to call [recreate](https://developer.android.com/reference/androidx/test/core/app/ActivityScenario.html#recreate()) on the activity scenario? Did you verify via step debugging that the activity is recreated? Please update test accordingly to make sure that it does go through the rotation as expected.